crates/swift-bridge-ir/src/bridged_type/shared_struct.rs

```diff
@@ -272,6 +314,341 @@ impl SharedStruct {
             }
         }
     }
+
+    pub fn tuple_from(types: &Vec<Type>) -> Option<Self> {
```

Comment:
Maybe `pub fn new_unnamed_with_types(types: &[Type]) -> Self` (but move it to `BuiltInTuple`)

Reply:
I got the following error. 

```
a value of type `&[syn::Type]` cannot be built from an iterator over elements of type `syn::Type`
the trait `FromIterator<syn::Type>` is not implemented for `&[syn::Type]`rustc[Click for full compiler diagnostic](bridged_type.rs)
bridged_type.rs(773, 73): required by a bound introduced by this call
iterator.rs(1832, 19): required by a bound in `collect`
```

So, I'll pass `Vec<Type>` to `new_unnamed_with_types`.